CHANGELOG.md

```diff
@@ -4,7 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.1.10] - 2019-08-18
+## [0.1.11] - 2019-08-25
+### Fixed
+- Always implement `std`-dependent traits for `std` targets even with
```

Comment:
It actually doesn't add the `std` traits, just the ones that had it in `0.1.8`. For example, Windows still won't have the trait enabled by default.

Reply:
Yes, you are right. Can you take a look at the new description?